protos/flyteidl/event/event.proto

```diff
@@ -93,6 +95,14 @@ message TaskNodeMetadata {
     core.CatalogMetadata catalog_key = 2;
 }
 
+// For dynamic workflow nodes we send information about the dynamic workflow definition that gets generated.
+message DynamicWorkflowNodeMetadata {
+    // id represents the unique identifier of the workflow.
+    core.Identifier id     = 1;
+
+    // Represents the compiled representation of the embedded dynamic workflow.
+    core.CompiledWorkflowClosure compiled_workflow = 2;
```

Comment:
is this what a regular workflow is also represented as?

Reply:
yes. https://github.com/flyteorg/flyteidl/blob/master/protos/flyteidl/admin/workflow.proto#L60